Nighttime-Operations/Observatory/DIMM/Start-up.rst

```diff
@@ -0,0 +1,151 @@
+ 
+
+Connect via telnet to the `tt-meteo`. 
+
+.. code-block:: bash
+   
+        dimm@dimm:~$  telnet 127.0.0.1 16301
+
+
+The prompt within OpenTPL, a plain-text client-server protocol, is ``AUTH OK 0 0`` Commands are prepended with a freely chosen command id (unsigned integer) Here increasing integers (1,2) were used. 
+More on this communication protocol can be found on the :ref:`DIMM general specs <DIMM-DIMM>`.
+
+The following two commands can be written once the prompt appears. 
+
+.. code-block:: bash
+    
+        1 set sky.temp=-30
+    
+        2 set sky.status=0
+
+Quit telnet, pressing simultaneously ``Ctrl`` and  ``]``, write ``quit`` and click Enter. 
+
+The image below shows how the interaction with the OpenTPL client looks like and the two commands required to manually configure `tt-meteo`.
+
+.. figure:: ./_static/tt-meteo.png
+    :name: tt-meteo prerequisite
+
+    Prerequisite: Weather data
+
+Now the DIMM is ready to start operations. Follow only one of the procedures below, either start the DIMM up in :ref:`automatic mode <Dimm_StartUp-Automatic>` or in 
+:ref:`manual mode <Dimm_StartUp-Manual>`
+
+.. _Dimm_StartUp-Automatic: 
+
+Starting up the DIMM in automatic mode
+======================================
+
+Make sure you have run and read the :ref:`prerequisites <Dimm_StartUp-Prerequisites>` regarding the weather data. 
+
+
+In this mode of operations, the DIMM `ameba` will automatically select the targets from the catalog. 
+To start an automatic DIMM observation, connect to `tt-master` where you will set ``ameba.mode=1``:
+
+.. code-block:: bash
+   
+        dimm@dimm:~$  telnet 127.0.0.1 16500
+
+
+and within this OpenTPL:
+
+.. code-block:: bash
+   
+        AUTH OK 0 0
+        1 set ameba.mode=1
+        
+
+Make sure the DIMM CSC is enabled and data is flowing to the EFD. 
+
+Once this mode is enabled, the DIMM will continue taking data until sunrise, and will run again the following night. You can manually stop the DIMM from doing so, setting the ``ameba.mode=0``. See shutdown procedure. 
+
+To monitor the status of the DIMM program, see :ref:`monitor the DIMM <Dimm-Monitor>`.
+
+.. _Dimm_StartUp-Manual:
+
+Starting the DIMM in manual mode
+====================================
+
+Make sure you have run and read the :ref:`prerequisites <Dimm_StartUp-Prerequisites>` regarding the weather data. 
+
+In this mode of operations, the user needs to choose the target and run the ``monitor_dimm_2.py`` python script available in the dimm home directory:
+
+.. note:: Substitute the following parameters in the command below. `name` is your target name. `RA` is expressed in hours, float; `dec` in deg, float. Keep `mag` and `color` in 0.0, otherwise, it fails.  
+
+.. code-block:: bash
+   
+        dimm@dimm:~$  python3 monitor_dimm_2.py manual --name HD4128 
+        --ra 0.7264917 --dec -17.9865833 --mag 0.0 --color 0.0 --sptype G9
+
+
+Make sure the DIMM CSC is enabled and data is flowing to the EFD. 
```

Comment:
Consider:
Verify the DIMM CSC is enabled and data is appearing in the EFD.

Reply:
👍